joomla/joomla-cms
# [com_tags] review config global default options

### Summary of Changes

This PR intends to normalize config global default options in com_tags.

The method used was:
1. Review cs of the config.xml file - also add missing default values
2. Check the current params in installation joomla.sql mysql
3. Add missing defaults to install sql

### Testing Instructions

0. Code review
1. Apply patch
2. Delete configuration.php
3. Install Joomla (no sample)
4. Create tag menu items and check all "Use Global"  have the global value.

### Documentation Changes Required

None.

### Added/changed defaults in the xml

changed:
- history_limit: 10 (was 5 - uniformize with articles)
- tag_list_orderby: c.core_title (was `title` - which doesn't exist)

added:
- tag_list_image: *empty*
- tag_list_show_item_description: 1 (the first option of the list)
- tag_list_item_maximum_characters: 0 (not sure about this one please confirm)
- all_tags_tag_maximum_characters: 0 (not sure about this one please confirm)

Note: also correct the parameter `all_tags_show_tag_descripion` by `all_tags_show_tag_description` this was nto working because the parameter was badly named in the xml files
